docs/src/main/sphinx/release/release-460.md

```diff
@@ -0,0 +1,52 @@
+# Release 460 (2 Oct 2024)
+
+## Atop connector
+
+* {{breaking}} Remove the Atop connector. ({issue}`23550`)
+
+## ClickHouse connector
+
+* Improve performance of listing columns. ({issue}`23429`)
+* Improve performance for queries comparing `varchar` columns. ({issue}`23558`) 
+* Improve performance for queries using `varchar` columns for `IN` comparisons. ({issue}`23581`)
+* Improve performance for queries with complex expression for `LIKE` clauses. ({issue}`23591`)
+
+## Delta Lake connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hive connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Hudi connector
+
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+
+## Iceberg connector
+
+* Add support for using an [Alluxio cluster as file system
+  cache](/object-storage/file-system-alluxio). ({issue}`21603`)
+* Add support for WASBS to [](/object-storage/file-system-azure). ({issue}`23548`)
+* Ensure table columns are cached in Glue even when table comment is too long. ({issue}`23483`)
+* Reduce planning time for queries on tables with a large number of structural
+  type fields. ({issue}`23451`)
+* Fix writing of bloom filter columns in Parquet files. ({issue}`22701`)
+
+## Oracle connector
+
+* Improve performance for queries casting `char` or `varchar` columns. ({issue}`22728`)
```

Comment:
Is this for casts between char <> varchar or something else?

Reply:
Original suggestion was

```
# Oracle
* Add support for cast project pushdown with Char and Varchar data type. ({issue}`22728`
```

and I think its any cast of char or varchar. I reworded ..

Reply:
I think the rewording I did is correct.

Reply:
> Is this for casts between char <> varchar or something else?

supporting cast pushdown when casting to `char` or `varchar`

```
char(m) -> char(n)
varchar(m) -> char(n)
```

```
varchar(m) -> varchar(n)
tinyint -> varchar(n)
smallint -> varchar(n)
int -> varchar(n)
bigint -> varchar(n)
decimal(p, q) -> varchar(n)
```

Reply:
oh.. the important thing is casting TO char or varchar.. will update